File: data/python/BASIC_AND_EXTENDED_EUCLIDEAN_ALGORITHMS_MAIN.py
if __name__ == '__main__':
    param = [
    (46,89,),
    (26,82,),
    (40,12,),
    (58,4,),
    (25,44,),
    (2,87,),
    (8,65,),
    (21,87,),
    (82,10,),
    (17,61,)
        ]
    n_success = 0
    print("class,mutant,test_index,result")
    for i, parameters_set in enumerate(param):
        try:
            print("BASIC_AND_EXTENDED_EUCLIDEAN_ALGORITHMS," + sys.argv[1] + "," + str(i) + "," + str(f_gold(*parameters_set)))
        except:
            print("BASIC_AND_EXTENDED_EUCLIDEAN_ALGORITHMS," + sys.argv[1] + "," + str(i) + ",EXCEPTION")

File: data/java/BASIC_AND_EXTENDED_EUCLIDEAN_ALGORITHMS_MAIN.java
public static void main(String args[]) throws IOException {
    int n_success = 0;
    List<Integer> param0 = new ArrayList<>();
    param0.add(46);
    param0.add(26);
    param0.add(40);
    param0.add(58);
    param0.add(25);
    param0.add(2);
    param0.add(8);
    param0.add(21);
    param0.add(82);
    param0.add(17);
    List<Integer> param1 = new ArrayList<>();
    param1.add(89);
    param1.add(82);
    param1.add(12);
    param1.add(4);
    param1.add(44);
    param1.add(87);
    param1.add(65);
    param1.add(87);
    param1.add(10);
    param1.add(61);
    StringBuilder builder = new StringBuilder();
    builder.append("class,mutant,test_index,result");
    FileWriter writer = new FileWriter(args[0]);
    for(int i = 0; i < param0.size(); ++i)
    {
        try {
	        if(f_filled(param0.get(i),param1.get(i)) == f_gold(param0.get(i),param1.get(i)))
	        {
                builder.append("BASIC_AND_EXTENDED_EUCLIDEAN_ALGORITHMS," + args[1] + "," + i + ",SUCCESS\n");
            } else {
                builder.append("BASIC_AND_EXTENDED_EUCLIDEAN_ALGORITHMS," + args[1] + "," + i + ",FAILURE\n");
            }
        } catch (Exception e) {
            builder.append("BASIC_AND_EXTENDED_EUCLIDEAN_ALGORITHMS," + args[1] + "," + i + ",EXCEPTION\n");
        }
    }
    writer.write(builder.toString());
    writer.close();
}
